gwas_sumstats_tools/schema/metadata.py

```diff
@@ -31,38 +31,45 @@ class CoordinateSystemEnum(str, Enum):
 
 
 class SampleMetadata(BaseModel):
+    sample_ancestry_category: List[str] = None
+    sample_ancestry: List[str] = None
+    sample_size: int = None
```

Comment:
Should these be `Optional` as their default values are `None`?

Reply:
According to the scheme, all studies should have sample_ancestry_category and sample_size. It is true that sample_ancestry is optional right now.

Reply:
I kept the SumStatsMetadata as same as the scheme defined in the https://github.com/EBISPOT/gwas-summary-statistics-standard/blob/master/schema/metadata-yamale-schema.yaml